srcpkgs/66-tools/template

```diff
@@ -1,7 +1,7 @@
 # Template file for '66-tools'
 pkgname=66-tools
```

Comment:
66-tools doesn't depend on oblibs, which I guess it's because it's statically linked into it. Do you want to look into doing it dynamically?

Reply:
You are correct, that is the upstream default, for both 66 and 66-tools - check the ldd output. I do not see a reason to change this behavior. oblibs is a small utility lib for 66 and 66-tools, it is not used outside of these projects.